src/index.js

```diff
@@ -1,43 +1,184 @@
 import './style.css';
+// import { update } from 'lodash';
 import Dots from './images/dots.png';
 import Enter from './images/enter.png';
+import Bin from './images/bin.png';
 import Refresh from './images/refresh.png';
+import Store from './StoreClass.js';
+import Task from './TaskClass.js';
+// import { remove } from 'lodash';
+// import { functionsIn, update } from 'lodash';
```

Comment:
- [ ] Dear @paulinagonzalezc, You did well implementing suggested changes from a previous code reviewer. Yet there is one main requirement you have missed at this point. I see you have commented on some codes on your code base. This is against good coding practice. Also will make your codebase a bit messy. Remember you can always go back in history (using your commit history) to check any code you have committed once upon a time.